source/reference/class/MongoDBClientBulkWrite.txt

```diff
@@ -0,0 +1,45 @@
+==============================
+MongoDB\\ClientBulkWrite Class
+==============================
+
+.. versionadded:: 2.1
+
+Definition
+----------
+
+.. phpclass:: MongoDB\ClientBulkWrite
+
+   This class allows you to assemble a bulk write command to pass to
```

Comment:
I learned that "allows" should be reserved to "permission", and words like "enable" should be used. But I might be wrong.

Reply:
We use both in many contexts, but I agree enables makes more sense here!